### PR TITLE
TK3.5: add service args to get-access-control

### PR DESCRIPTION
**Pull request by frenchy64**
Related https://github.com/threatgrid/iroh/issues/3986

<a name="qa">[§](#qa)</a> QA
============================

No QA is needed.

<a name="release-notes">[§](#release-notes)</a> Release Notes
=============================================================

```
intern: TK3.5: add service args to get-access-control
```

<a name="squashed-commits">[§](#squashed-commits)</a> Squashed Commits
======================================================================

### CODE REVIEW

File: test/ctia/domain/access_control_test.clj [hunk 1]
```diff
@@ -83,110 +84,126 @@
...
-       (test-matching-user "red" sut/allow-read? true?)
-       (test-matching-group "red" sut/allow-read? false?)
-       (test-no-group "red" sut/allow-read? false?)
-       (test-user-group-mismatch "red" sut/allow-read? false?)
-       (test-authorized_users-mismatch "red" sut/allow-read? false?)
-       (test-authorized_groups-mismatch "red" sut/allow-read? false?)
-       (test-authorized_users-match "red" sut/allow-read? true?)
-       (test-authorized_groups-match "red" sut/allow-read? true?))))
+   (fn [get-in-config]
+    (let [allow-read? #(sut/allow-read? %1 %2 get-in-config)]
```
Comment (vemv) [inline test/ctia/domain/access_control_test.clj:197]: fn and let have 2-space indentation

**Review by vemv — APPROVED** [review]
lgtm!

File: test/ctia/domain/access_control_test.clj (hunk 1 above)
Comment (frenchy64) [reply test/ctia/domain/access_control_test.clj:197]: Indeed. A failed attempt at a compact diff, registers as one chunk with all the `sut/` deletions. Will fix.

Comment (frenchy64) [reply test/ctia/domain/access_control_test.clj:197]: The weird indentation does give a nicer diff on my machine (just the `sut/`s are deleted), but it's useless on GitHub some reason.

**Pull request merged**